examples/multiple_services/README.md

```diff
@@ -1,4 +1,4 @@
-# Multiple Services Binding Example
+# Binding Multiple Services to an application
```

Comment:
The example contains use of OperatorSource (which is obsolete) as well as CatalogSource (which is correct), please remove the OperatorSource (https://github.com/pratikjagrut/service-binding-operator/pull/3)

Reply:
@pmacik Done.